components/director/pkg/formationconstraint/join_point_details.go

```diff
@@ -74,12 +74,11 @@ func (d *UnassignFormationOperationDetails) GetMatchingDetails() MatchingDetails
 // GenerateFormationAssignmentNotificationOperationDetails contains details applicable to generate formation assignment notifications join point
 type GenerateFormationAssignmentNotificationOperationDetails struct {
 	Operation             model.FormationOperation
-	FormationID           string
-	FormationTemplateID   string
```

Comment:
Should this be removed?

Reply:
It's related to the other comment, check the answer of it.